api/v1alpha1/gatewaylbconfiguration_types.go

```diff
@@ -23,6 +23,10 @@ type GatewayLBConfigurationSpec struct {
 	// +optional
 	GatewayVmssProfile `json:"gatewayVmssProfile,omitempty"`
 
+	// Profile of the gateway VM to apply the gateway configuration.
+	// +optional
+	GatewayVmProfile `json:"gatewayVmProfile,omitempty"`
```

Comment:
wondering if we should have this be `gatewayPoolProfile` or something and eventually deprecate the specific `Vmss` Profile.